octobot_trading/enums.py

```diff
@@ -193,6 +198,7 @@ class ExchangePositionCCXTColumns(enum.Enum):
 
 class ExchangeConstantsFundingColumns(enum.Enum):
     SYMBOL = "symbol"
+    TIMESTAMP = "timestamp"
```

Comment:
Should we care about funding timestamp? As it can't change until next funding time

Reply:
I guess we should care cause it also works with funding history